### PR TITLE
Request to add a new person "Mohammad-Ebadi"

### PR DESCRIPTION
**Pull request by mohammad-ebadi**

### CODE REVIEW

**Review by javadebadi — CHANGES_REQUESTED** [review]
@Mohammad-develope 
Please add your interests. What fields or subjects are you interested in?

File: README.md [hunk 1]
```diff
@@ -32,3 +32,4 @@ Saeed | Sarkarati | **Photonics/Physics, PhD** | Physics, Programming, Teaching,
 Jafar | Khaksar | **Economics, MSc student** | Quantitative Economics, Econometrics, Data Science, Julia 
 Mobin | Moradi | **Physics, BSc** | Physics, Mathematics
 Mehdi | Mirzaie | **Mathematics. Ph.D.** | Data Science, Bioinformatics
+Mohammad | Ebadi | None | None
```
Comment (javadebadi) [inline README.md]: Please add your interests in the fourth column of the table

File: README.md [hunk 2]
```diff
@@ -32,4 +32,4 @@ Saeed | Sarkarati | **Photonics/Physics, PhD** | Physics, Programming, Teaching,
 Jafar | Khaksar | **Economics, MSc student** | Quantitative Economics, Econometrics, Data Science, Julia 
 Mobin | Moradi | **Physics, BSc** | Physics, Mathematics
 Mehdi | Mirzaie | **Mathematics. Ph.D.** | Data Science, Bioinformatics
-Mohammad | Ebadi | None | None
+Mohammad | Ebadi | **Associate degree**	 | Programming , Web designing , Cybersecurity , Network
```
Comment (javadebadi) [inline README.md]: @Mohammad-develope
Recommended Changes:
Web designing to Web Desgin
Network to Networking
degree to Degree

Comment (javadebadi) [reply README.md:35]: @Mohammad-develope 
Also change the "degree" to "Degree"

Comment (mohammad-ebadi) [reply README.md]: Sure.

File: README.md [hunk 3]
```diff
@@ -32,4 +32,4 @@ Saeed | Sarkarati | **Photonics/Physics, PhD** | Physics, Programming, Teaching,
 Jafar | Khaksar | **Economics, MSc student** | Quantitative Economics, Econometrics, Data Science, Julia 
 Mobin | Moradi | **Physics, BSc** | Physics, Mathematics
 Mehdi | Mirzaie | **Mathematics. Ph.D.** | Data Science, Bioinformatics
-Mohammad | Ebadi | **Associate degree**	 | Programming , Web designing , Cybersecurity , Network 
+Mohammad | Ebadi | **Associate degree**	 | Programming , Web design , Cybersecurity , Networking
```
Comment (javadebadi) [inline README.md]: @Mohammad-develope 

Please transform the "design" to "Design" with first letter capitalized
Also for the "degree" -> "Degree"

File: README.md [hunk 4]
```diff
@@ -32,4 +32,4 @@ Saeed | Sarkarati | **Photonics/Physics, PhD** | Physics, Programming, Teaching,
 Jafar | Khaksar | **Economics, MSc student** | Quantitative Economics, Econometrics, Data Science, Julia 
 Mobin | Moradi | **Physics, BSc** | Physics, Mathematics
 Mehdi | Mirzaie | **Mathematics. Ph.D.** | Data Science, Bioinformatics
-Mohammad | Ebadi | **Associate degree**	 | Programming , Web design , Cybersecurity , Networking
+Mohammad | Ebadi | **Diploma Degree**	 | Programming , Web design , Cybersecurity , Networking
```
Comment (javadebadi) [inline README.md]: Great.

@Mohammad-develope 
Please also change the "Web design" -> "Web Design". Use Captial letter for the first letter of the word "design"

**Comment by javadebadi** [comment]
Thank you @Mohammad-develope 
I will be happy to merge your changes.

**Comment by mohammad-ebadi** [comment]
> Thank you @Mohammad-develope I will be happy to merge your changes.

I'll be happy too  Javad.

**Pull request merged**